test/valgrind/valgrindTestSuite.sh

```diff
@@ -259,64 +241,6 @@ function add()
 
 
 
-# -----------------------------------------------------------------------------
-#
-# Start broker
-#
-function brokerStart()
```

Comment:
Unused function (I guess it comes from the time .vtest files were used but they were removed time ago).

NTC (just informative)